Items.cs

```diff
@@ -890,10 +902,20 @@ public class Items
                 "Map: Romani Ranch",
                 "Map: Great Bay",
                 "Map: Stone Tower",
-                "Goron Racetrack Grotto"
+                "Goron Racetrack Grotto",
+                "One Mask",

```

Comment:
What do these mean? "One Mask", "Two Masks" etc.

Reply:
on the moon, the requirements for entering the sub dungeons are to give up 1, 2, 3, and 4 masks at a time